configure/accessibility-options.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
`options` isn't used in any other page, so this filename should probably just be `accessibility.md`